beaconclient/multi_beacon_client.go

```diff
@@ -29,7 +29,11 @@ const (
 )
 
 func (b BroadcastMode) String() string {
-	return [...]string{"gossip", "consensus", "consensus_and_equivocation"}[b]
+	broadcastModeStrings := [...]string{"gossip", "consensus", "consensus_and_equivocation"}
+	if int(b) >= len(broadcastModeStrings) {
+		return "invalid broadcast mode value"
```

Comment:
hmmm, not sure this is a good pattern, returning an error as default string as if nothing was wrong... 

i think there's probably a better way to do this whole thing 🤔 like defining the broadcastmodes as string consts?

```go
type BroadcastMode string

const (
    BroadcastModeGossip BroadcastMode = "gossip"
    BroadcastModeConsensus      = "consensus"
    BroadcastModeConsensusAndEquivocation      = "consensus_and_equivocation"
)
```